src/platform/src/common/features/Checklist/utils/steps.js

```
@@ -1,146 +1,146 @@
+    // Find corresponding checklist item, or create an empty object
+    const checklistItem = checklistItems[index] || {};
+
+    return {
+      ...step,
+      _id: checklistItem._id,
+      status: checklistItem.status || 'not started',
+      completed: checklistItem.completed || false,
+      videoProgress: checklistItem.videoProgress || 0,
+      completionDate: checklistItem.completionDate || null,
+      title:
+        checklistItem.title && checklistItem.title.trim()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)